webcompat/helpers.py

```diff
@@ -414,7 +414,7 @@ def extract_url(issue_body):
     URL in webcompat.com bugs follow this pattern:
     **URL**: https://example.com/foobar
     """
-    url_pattern = re.compile('\*\*URL\*\*\: (.+)\n')
+    url_pattern = re.compile(r'\*\*URL\*\*\: (.+)')
```

Comment:
just explaining here the why. This was W605 invalid escape sequence ‘x’
The explanation is https://lintlyci.github.io/Flake8Rules/rules/W605.html

https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals

> Both string and bytes literals may optionally be prefixed with a letter 'r' or 'R'; such strings are called raw strings and treat backslashes as literal characters. As a result, in string literals, '\U' and '\u' escapes in raw strings are not treated specially. Given that Python 2.x’s raw unicode literals behave differently than Python 3.x’s the 'ur' syntax is not supported.